pyscriptjs/src/components/pyscript.ts

```diff
@@ -237,6 +237,12 @@ export async function mountElements(interpreter: InterpreterClient) {
     const matches: NodeListOf<HTMLElement> = document.querySelectorAll('[py-mount]');
     logger.info(`py-mount: found ${matches.length} elements`);
 
+    if (matches.length > 0) {
+        const deprecationMessage =
+            'The "py-mount" attribute is deprecated. Please add references to HTML Elements manually in your script.';
+        createDeprecationWarning(deprecationMessage, 'py-mount');
+    }
```

Comment:
please add a comment to specify in which release the deprecation was added.
It's a bit weird because we don't know yet what the release number will be, maybe we can write something like:
- `deprecated after 2023.03.01`
- `deprecated from version > 2023.03.1`
- `last non-deprecated version: 2023.03.1`

Reply:
Good thinking; added a comment at the deprecation warning itself and in the test for it, though the latter may be overkill.

Reply:
thanks!